pkg/kubelet/apis/config/validation/validation_test.go

```diff
@@ -14,22 +14,25 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package validation
+package validation_test
```

Comment:
s/validation_test/validation/g

Reply:
I thought that was one of the best practices to remove test files whenever we don't have to call unexported methods/functions from a test file but no? I found some test files with the `_test` suffix in this project but it does not seem that popular though...

Ref: https://pkg.go.dev/cmd/go@master#hdr-Test_packages
> Test files that declare a package with the suffix "_test" will be compiled as a separate package, and then linked and run with the main test binary.

Reply:
Kubernetes doesn't necessarily follow all golang conventions, we do this very inconsistently in kubelet:

```
ehashman@fedora:~/src/k8s$ git grep 'package .\+_test' pkg/kubelet/
pkg/kubelet/container/runtime_cache_test.go:package container_test
pkg/kubelet/envvars/envvars_test.go:package envvars_test
pkg/kubelet/kubelet_dockerless_test.go:package kubelet_test
pkg/kubelet/runtimeclass/runtimeclass_manager_test.go:package runtimeclass_test
```

We can leave this as is or not, I don't think it really matters.